validator/accounts/v2/wallet_create_test.go

```diff
@@ -217,6 +217,38 @@ func TestCreateWallet_Derived(t *testing.T) {
 	assert.DeepEqual(t, derived.DefaultKeymanagerOpts(), cfg)
 }
 
+// TestCreateWallet_IfExistingWallet checks for expected error if trying to create a wallet where they is already one.
+func TestCreateWallet_IfExistingWallet(t *testing.T) {
+	walletDir, passwordsDir, passwordFile := setupWalletAndPasswordsDir(t)
+	cliCtx := setupWalletCtx(t, &testWalletConfig{
+		walletDir:          walletDir,
+		passwordsDir:       passwordsDir,
+		walletPasswordFile: passwordFile,
+		keymanagerKind:     v2keymanager.Derived,
+	})
+
+	// We attempt to create the wallet.
+	_, err := CreateAndSaveWalletCli(cliCtx)
+	require.NoError(t, err)
+
+	// We attempt to create another wallet of the same type at the same location.  We expect an error.
+	_, err = CreateAndSaveWalletCli(cliCtx)
+	require.NotEqual(t, nil, err,
```

Comment:
Please use `require.ErrorContains` and verify the error message.

Reply:
I think the test in its current will pass even with the old code.

Reply:
Hey I made a branch from current main that only has the test:  https://github.com/dv8silencer/prysm/tree/dv8-PR-7268-Regression  This is the diff vs current master:  https://github.com/prysmaticlabs/prysm/compare/master...dv8silencer:dv8-PR-7268-Regression?expand=1

When running:
```
root@112af41569a9:~/prysm/validator/accounts/v2# go test ./... --tags=blst_enabled -v | grep TestCreateWallet_WalletAlreadyExists
=== RUN   TestCreateWallet_WalletAlreadyExists
--- FAIL: TestCreateWallet_WalletAlreadyExists (0.01s)
```

When running PR branch:
```
root@112af41569a9:~/prysm/validator/accounts/v2# go test ./... --tags=blst_enabled -v | grep TestCreateWallet_WalletAlreadyExists
=== RUN   TestCreateWallet_WalletAlreadyExists
--- PASS: TestCreateWallet_WalletAlreadyExists (0.01s)
```